### ISSUE TITLE
Calling Chart.plot.line method with all 0 y_column raises OverflowError

### ISSUE THREAD

**Issue by fabioconcina** [spotify/chartify#62]
type: bug

**Environment**:

- Chartify version(s): 2.3.5
- Operating System(s): MacOS 10.14.2
- Python version(s): 3.6.5

**What happened**:
Calling line method with all 0 y_column raises OverflowError

**What you expected to happen**:
Return an instance of chart

**How to reproduce it (as minimally and precisely as possible)**:
```python
import chartify
import pandas as pd

df = pd.DataFrame(data={'col1': [1, 2, 3], 'col2': [0, 0, 0]})

ch = chartify.Chart()
ch.plot.line(data_frame=df,
             x_column='col1',
             y_column='col2')
```

**Comment by cphalpert** [comment]
Thanks for reporting the bug!

**Comment by cphalpert** [comment]
Fixed in #39
